packages/components/src/Form/Inputs/Combobox/ComboboxOption.tsx

```diff
@@ -135,7 +139,22 @@ export const ComboboxOptionWrapper = forwardRef(
   )
 )
 
-ComboboxOptionWrapper.displayName = 'ComboboxOptionWrapper'
+ComboboxOptionWrapperInternal.displayName = 'ComboboxOptionWrapper'
+
+export const ComboboxOptionWrapper = styled(ComboboxOptionWrapperInternal)`
+  background-color: ${({ isSelected, theme }) =>
+    isSelected && theme.colors.keySubtle};
+
+  &:hover {
+    background-color: ${({ isSelected, theme }) =>
+      isSelected ? theme.colors.keyAccent : theme.colors.ui1};
+  }
+
+  /** Essentially on hover and when "selected" */
+  /* &[aria-selected='true'] {
+    background-color: ${(props) => props.theme.colors.ui1};
+  } */
```

Comment:
Right now keyboard nav doesn't get any feedback on the "active" option. If you make this change, I think it looks good for both keyboard nav and hover, although when you first open a `Select` that has a current value, it'll be `keyAccent` because it is both "active" and "selected". If that's not what we want, we'll probably have to think of something fancy here.
```suggestion
  &[aria-selected='true'] {
    background-color: ${({ isSelected, theme }) =>
      isSelected ? theme.colors.keyAccent : theme.colors.ui1};
  }
```

Reply:
👍 Good catch. 

Regarding the opening a `Select` with a current value and that value having a `keyAccent` background, I think I'd defer to @jhardy.

Reply:
Also you're not blind! I imagine those look really similar because of the opacity we apply to `TreeItem` backgrounds. We do this because we don't want to blot out the optional vertical `border`:
<img width="229" alt="Screen Shot 2020-12-11 at 2 55 35 PM" src="https://user-images.githubusercontent.com/16812885/101962404-ef54be00-3bc0-11eb-9834-bdb439e1543b.png">

Reply:
Spoke to @jhardy and he approves of this!